src/main/resources/interface-specification.yml

```diff
@@ -8311,6 +8311,16 @@ paths:
           schema:
             type: string
             format: uuid
+        - in: query
+          name: relationshipIds
```

Comment:
[same](https://github.com/pagopa/interop-be-authorization-process/pull/225#discussion_r1331140725), I think this feature is intended on the retrieve, instead of the creation